frontend/src/components/MeetingRoom/ReportIssue/FeedbackForm/FeedbackForm.tsx

```diff
@@ -60,8 +60,9 @@ const FeedbackForm = ({
   onFileSelect,
 }: FeedbackFormType): ReactElement => {
   const isSmallViewport = useIsSmallViewport();
-  const heightClass = '@apply h-[calc(100dvh_-_240px)]';
-  const widthClass = isSmallViewport ? '@apply w-[calc(100dvw_-_48px)]' : '';
+  // Account for: 64px panel header + 80px toolbar if small viewport or 96px toolbar if normal viewport + (40px submit button + 24px submit button margin)
```

Comment:
IMO, if this were a calculation, as opposed to a comment, the correlation might be more easily inferred.